### PR TITLE
Bug 834462 - Add version and build ID information to the update dialog

### PR DESCRIPTION
**Pull request by gabrielesvelto**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0160740248a57d68f7f76c93d34773846472c7c8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ea1ca0a8746b67ad536844eebfe9247d35a82919

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=11b15b0fa545da73753f9ce729c089f9c5d01efd

File: apps/system/js/update_manager.js [hunk 1]
```diff
@@ -410,6 +410,32 @@
...
+          _localize(buildId, 'build-id', { buildid: updatable.buildID });
+          listItem.appendChild(buildId);
+        }
+
+        if (updatable.detailsURL && (updatable.detailsURL !== 'about:blank')) {
+          var detailsUrl = document.createElement('a');
+          _localize(detailsUrl, 'view-release-notes');
+          listItem.appendChild(detailsUrl);
+          listItem.onclick = (function(event) {
+            event.preventDefault();
```
Comment (etiennesegonzac) [inline apps/system/js/update_manager.js]: can you cover this with a test?
(asserting `defaultPrevented` and using `MockMozActivity` to assert that we launched the correct activity)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5598484e259b66639768f315bc98d43c16c5aff5

**Pull request merged**